src/main/java/com/cs525/reversi/models/ReversiGameBuilder.java

```diff
@@ -0,0 +1,65 @@
+package com.cs525.reversi.models;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+
+public class ReversiGameBuilder implements GameBuilder {
+
+	private Game reversiGame = null;
+
+	public ReversiGameBuilder() {
+		this.reversiGame = new Game();
+	}
+
+	@Override
+	public ReversiGameBuilder buildPlayerOne(User playerOne) {
+		this.reversiGame.setPlayer1(playerOne);
+		return this;
+
+	}
+
+	@Override
+	public ReversiGameBuilder buildPlayerTwo(User playerTwo) {
+		this.reversiGame.setPlayer2(playerTwo);
+		return this;
+
+
+	}
+    @Override
+	 public ReversiGameBuilder buildGameStatus(GameStatus gameStatus) {
+		 this.reversiGame.setStatus(gameStatus);
+		 return this;
+	 }
+    @Override
+	 public ReversiGameBuilder buildGameUUID() {
+		 this.reversiGame.setUuid(UUID.randomUUID());
+		 return this;
+	 }
+    @Override
+
+	 public ReversiGameBuilder buildBoardGame() {
+		 this.reversiGame.setRows(generateBoard());
+		 return this;
+	 }
+    
+	 private List<MatrixRow> generateBoard() {
+			List<MatrixRow> rows = new ArrayList<MatrixRow>();
+
+			for (int i = 1; i <= 8; i++) {
+				MatrixRow row = new MatrixRow();
+				List<String> cells = new ArrayList<String>();
+				for (int j = 1; j <= 8; j++) {
+						cells.add("");
+				}
+				row.setCells(cells);
```

Comment:
@MustafaMosad `List<String>` can't be set to `cells`

Reply:
@natnafel  this should be list <CellValue> you should applied